aviary/subsystems/propulsion/motor/model/motor_mission.py

```diff
@@ -0,0 +1,133 @@
+        ivc.add_output(Dynamic.Mission.THRUST, val=np.zeros(n), units='N')
+        ivc.add_output(Dynamic.Mission.THRUST_MAX, val=np.zeros(n), units='N')
+        ivc.add_output(Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE,
+                       val=np.zeros(n), units='kg/s')
+        ivc.add_output(Dynamic.Mission.NOX_RATE, val=np.zeros(n), units='kg/s')
+
+        ivc.add_output('max_throttle', val=np.ones(n), units='unitless')
+
+        self.add_subsystem('ivc', ivc, promotes=['*'])
+
+        motor_group = om.Group()
+
+        motor_group.add_subsystem('motor_map', MotorMap(num_nodes=n),
+                                  promotes_inputs=[Dynamic.Mission.THROTTLE,
+                                                   Aircraft.Engine.SCALE_FACTOR,
+                                                   Dynamic.Mission.RPM],
+                                  promotes_outputs=[(Dynamic.Mission.TORQUE, 'motor_torque'),
+                                                    'motor_efficiency'])
+
+        motor_group.add_subsystem('power_comp',
+                                  om.ExecComp('shaft_power = torque * pi * RPM / 30',
+                                              shaft_power={
+                                                  'val': np.ones(n), 'units': 'kW'},
+                                              torque={'val': np.ones(
+                                                  n), 'units': 'kN*m'},
+                                              RPM={'val': np.ones(n), 'units': 'rpm'}),  # fixed RPM system
+                                  promotes_inputs=[('torque', 'motor_torque'),
+                                                   ('RPM', Dynamic.Mission.RPM)],
+                                  promotes_outputs=[('shaft_power', Dynamic.Mission.SHAFT_POWER)])
+
+        motor_group.add_subsystem('energy_comp',
+                                  om.ExecComp('power_elec = shaft_power / efficiency',
+                                              shaft_power={
+                                                  'val': np.ones(n), 'units': 'kW'},
+                                              power_elec={'val': np.ones(
+                                                  n), 'units': 'kW'},
+                                              efficiency={'val': np.ones(n), 'units': 'unitless'}),
+                                  promotes_inputs=[
+                                      #   ('shaft_power', Dynamic.Mission.SHAFT_POWER),
+                                      ('efficiency', 'motor_efficiency')],
+                                  promotes_outputs=[('power_elec', Dynamic.Mission.ELECTRIC_POWER_IN)])
+
+        motor_group.connect(Dynamic.Mission.SHAFT_POWER, 'energy_comp.shaft_power')
+
+        # TODO Gearbox needs to be its own component separate from motor
+        # is this needed???
+        # this may be already covered by throttle constraints being from 0 - 1
+        # motor_group.add_subsystem('torque_con',
```

Comment:
That old gearbox issue again. Did we agree that it needs to be completely separate from the motor model? If so all this code can be removed.